src/pytesmo/validation_framework/upscaling.py

```diff
@@ -221,6 +221,7 @@ def temporal_match(
             combined_dropna=combined_dropna,
             checkna=True,
         )
+        matched.dropna(axis="columns", how="all", inplace=True)
```

Comment:
Did `combined_dropna` not work as expected?

Reply:
The thing is that it only works on rows but this is to eliminate a whole column